Documentation/CrossPlatform/UsingARFoundation.md

```diff
@@ -10,15 +10,22 @@
 
     | **Android** | **iOS** | Comments |
     | --- | --- | --- |
-    | AR Foundation  <br/> Version: 1.5.0 - preview 6 | AR Foundation  <br/> Version: 1.5.0 - preview 6 | For Unity 2018.4, this package is included as a preview. To view package: Window > Package Manager > Advanced > Show Preview Packages|
+    | AR Foundation  <br/> Version: 1.5.0 - preview 7 | AR Foundation  <br/> Version: 1.5.0 - preview 7 | For Unity 2018.4, this package is included as a preview. To view package: Window > Package Manager > Advanced > Show Preview Packages|
```

Comment:
>Version: 1.5.0 - preview 7

@CDiaz-MS @davidkline-ms I feel like I remember we had been running into issues between preview 6 and preview 7?

Reply:
I remember that the preview version did have an impact on iOS testing.  

@Kent1LG  Has the AR Foundation preview 7 version been tested on both Android and iOS?

Reply:
Hi Kurtis and Catherine ! Thank you for this point, I didn't try IOS yet, I'm sorry. This is a matter of time, I'm gonna handle this tomorrow (French timing ahah) as soon as possible.
I keep you all in touch!

Reply:
Hi again, so, I just tried. Unfortunatly ArFoundation can't be use in 1.5 preview 7 versions. I tried preview 6 instead on both and this works fine.